### ISSUE TITLE
upload failed

### ISSUE THREAD

**Issue by boseann** [mayeaux/generate-subtitles#63]
I'm having trouble uploading audio files, the upload stops when almost done and it says files may be corrupt.
only today i started using the website, it worked for few times and then it stoped working, the files are m4a download with yt-dl , but i even tried an mp3 music file and still it didn't work.

**Comment by josephkolawole** [comment]
I have the same issue

**Comment by AuroraMartell** [comment]
I have the same problem - and I've tried on both Windows and Mac

**Comment by lyqht** [comment]
Same, getting the corrupted files error. 
Tried ogg/mp3 files. For the mp3 files that i uploaded, they work with whisper model from another API, so the files are probably not corrupted. The error message be a default error message. Please help us to look into it, thank you!

**Comment by niradar** [comment]
Same problem

**Comment by 300Sven** [comment]
I have the same problem and in doing some googling to find out if the developer had made any announcements re: the status of freesubtitles.ai on other platforms, I discovered that he is/was a fairly prominent white supremacist. Pretty weird.

(This is not a doxxing: Anthony has done essentially nothing to hide his identity, which is weirdly admirable)

**Comment by ghost** [comment]
Same problem and am getting no response to my queries either online or by email. I'm wondering whether it's time to talk to my credit card company about getting my money back? I'll be very sorry to see this service go, but this prolonged silence isn't promising.

**Comment by saschafiedler** [comment]
Same problem persists.

**Comment by ghost** [comment]
Yes, I tried again yesterday with the same result. It would be good to know whether there is reason to hope for a resolution, or if we should be concerned about Anthony...

**Comment by AuroraMartell** [comment]
The site works now.

**Comment by gbonnema** [comment]
That is real good news! Thank you, for telling us!!

**Comment by AuroraMartell** [comment]
> That is real good news! Thank you, for telling us!!

You're welcome!

**Comment by YHADJRABIA** [comment]
There seems to be an issue again, upload fails between 97% to 99%

**Comment by asheriif** [comment]
Yeah it's broken again.

**Comment by ratj** [comment]
There seems to be an issue. Upload fails between 97%

**Comment by roastme** [comment]
PSA: use https://github.com/openai/whisper or self host the site
